.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## This PR
+
+- [ ] Ticks this box and Closes #100000
+
+
+## Submission criteria
+
+Please review the below criteria before asking for a review.
+
+### Fulfilled the scope of this repo?
+
+- [ ] The component cannot be implemented by styling a MUI component via the theme
+- [ ] The component implements an element of the [DHI Design Guidelines](https://www.figma.com/file/pSfX5GNsa6xhKGbi3DWQtn/DHI-Official-Guidelines) or is otherwise generic enough in functionality and close enough to the DHI CVI that it is likely to find reuse in other projects
+
+### Completness
+
+- [ ] Story included
+
+### Picked the right package?
+
+- [ ] Just fixing existing stuff
+- [ ] Labs package for new submissions that fulfil the above criteria
+- [ ] Core package for selected components submitted by the SPOPX team
```

Comment:
Unsure about the last checkbox, shouldn't the rule be something like, is the component stable enough in the lab, tested, and used within our apps such that it's ready to be moved to the core?

Not sure if it's related to any department, it's DHI's react-components and I guess that a way to move components is decided in a meeting with all the water warriors bosses ⚔️

I believe that the SPOPX components should live in a specific folder since they are not related to DHI, but rather to solutions? - I might be speaking out of nowhere, but this is my understanding so far about these comps

Reply:
Agree, core is just for components that are considered consistent long-term high-value and tested enough to not have to have breaking changes later

Reply:
Yes, this point very much needs discussion. To me, it is unclear where the SPOPX team should in future place components like the ones currently in the "core" package. I guess those components will not necessarily go through the regular lifecycle of lab first, core (maybe) later, based on maturity and use frequency. Also, we do not want to disrupt their use of this repo too much.

Maybe I will just leave this point open (and remove that decision check box section) and we can discuss this with them later.